charts/kong/templates/tests/test-jobs.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if  .Values.deployment.testJobs.enabled }}
```

Comment:
It seems like rather than adding a new boolean, we could just leave the existing `test.enabled` boolean and check it conditionally here rather than having a separate one. I'm not sure I'm seeing a situation where you would want one, but not the other? :thinking: Let me know your thoughts

Reply:
@shaneutt That's how I initially had it, but I went with separating them because I noticed ```.Values.deployment.tests.enabled``` defaulted to ```false``` and the test jobs were enabled by default. I'm with you though, I think they should use the same condition. The question though is what should they default to?

Reply:
It should remain defaulted to `false`. We set it to `true` in the test configs that need it already. Currently these are effectively default `false` when using `helm install` because they have the test hook annotation, and Helm strips them out after templating.

I curse the ecosystem for not fully evaluating if the tools they were using were fit for purpose and do still think the burden of handling this should be upstream in the CD systems, but if we already have this flag, expanding its reach is fine.

Reply:
Updated my pr so it's in sync with main. Appreciate you guys entertaining this change.